lib/json/module_declarations/export_declaration.ts

```diff
@@ -12,7 +12,7 @@ export class ExportDeclaration extends Node {
     super(node, ConvertedSyntaxKind.ExportDeclaration);
 
     if (node.exportClause) {
-      this.exportClause = new NamedExports(node.exportClause);
+      this.exportClause = new NamedExports(node.exportClause as tsNamedExports);
```

Comment:
This is ugly, but no amount of `node.exportClause instanceof NamedExports` or similar did the trick :/